Packs/PrismaSaasSecurity/Integrations/SaasSecurityEventCollector/SaasSecurityEventCollector_description.md

```diff
@@ -0,0 +1,29 @@
+
+## Limitations
```

Comment:
Also, let's add here how a user should get the client secret and id

Reply:
it already exists in the description file, see below.